### ISSUE TITLE
Send Files

### ISSUE THREAD

**Issue by TheUntraceable** [EpikHost/EpikCord.py#57]
### Summary

You should be able to send files.

### The Problem

You can't send files with `Messageable.send`

### The Ideal Solution

To be able to send files.

### Checklist

- [X] This bug was found on the latest version of EpikCord
- [X] Full details of the requested feature have been provided/shown

### Additional Context

We need to implement this before 1.0.

**Comment by TheUntraceable** [comment]
You need to turn it into FormData, I don't know your method, I don't know much about Images and Bytes.

**Comment by Jerry-py** [comment]
I don't think this is resolved because we just implemented but we can't send it or we can?

**Comment by Jerry-py** [comment]
@nav-github01001

**Comment by TheUntraceable** [comment]
We could? How?

**Comment by EmreTech** [comment]
@TheUntraceable why did you not close this issue? There was a PR that clearly resolved this feature request.

**Comment by TheUntraceable** [comment]
I checked the source code for `.send`, it doesn't account for files/attachments
